be/src/io/cache/block_file_cache.cpp

```diff
@@ -1872,6 +1877,72 @@ void BlockFileCache::run_background_gc() {
     }
 }
 
+void BlockFileCache::run_background_lru_dump() {
+    int64_t interval_time_seconds = 60; // Dump every minute
```

Comment:
Actually, we should use num of evict and interval to control.

Reply:
will do in following optimization